sql/src/main/java/io/crate/metadata/sys/SysJobsLogTableInfo.java

```diff
@@ -45,6 +45,7 @@
 
     public static class Columns {
         public static final ColumnIdent ID = new ColumnIdent("id");
+        public static final ColumnIdent USER = new ColumnIdent("user");
```

Comment:
i think it would be better to call the column "username", since this does not conflict with the built in function "user", so it is not required to quote it when selecting and also is more explicit